consensus/polybft/state_sync_manager.go

```diff
@@ -22,8 +22,10 @@ import (
 )
 
 const (
-	// number of stateSyncEvents to be processed before a commitment message can be created and gossiped
-	stateSyncCommitmentSize = 10
+	// maximum number of stateSyncEvents that a commitment can have
+	maxCommitmentSize = 10
+	// minimum number of stateSyncEvents that a commitment can have
+	minCommitmentSize = 2
```

Comment:
Can you add a comment on why did we choose this number?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1083/commits/5536eee6702ed7cf4159803120aa141c8d1bfce7